src/lib/objects.js

```diff
@@ -0,0 +1,62 @@
+const createPerson = (name, age) => {
+  return {
+    name: name,
+    age: age
+  };
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  return property in object;
+};
+
+const isOver65 = person => {
+  return person.age > 65;
+};
+
+const getAges = people => {
+  return people.map(person => person.age);
+};
+
+const findByName = (name, people) => {
+  return people.find(person => person.name === name);
+};
+
+const findHondas = cars => {
+  return cars.filter(car => car.manufacturer === 'Honda');
+};
+
+const averageAge = people => {
+  const totalAge = people.reduce((sum, person) => sum + person.age, 0);
+  return totalAge / people.length;
+};
+
+const createTalkingPerson = (name, age) => {
+  return {
+    name: name,
+    age: age,
+    introduce: (friendName) => {
+      return `Hi ${friendName}, my name is ${name} and I am ${age}!`;
+    }
```

Comment:
As above you can use the shorthand for lines 43/44, and if you remove the brackets around the argument for the function it will stop complaining there too:
```
    name,
    age,
    introduce: friendName => {
      return `Hi ${friendName}, my name is ${name} and I am ${age}!`;
    },
```